plugins/woocommerce-blocks/tests/e2e/tests/templates/template-customization.block_theme.spec.ts

```diff
@@ -41,11 +42,16 @@ test.describe( 'Template customization', () => {
 				await editor.saveSiteEditorEntities( {
 					isOnlyCurrentEntityDirty: true,
 				} );
+
+				const name =
```

Comment:
The name option in getByRole supports Regex. I think you should be able to express this locator without needing the WordPress version.